src/redux/reducers.js

```diff
@@ -0,0 +1,17 @@
+import { combineReducers } from 'redux';
+import { FETCH_RANDOM_GREETING_SUCCESS } from './actions';
+
+const greetingReducer = (state = '', action) => {
+  switch (action.type) {
+    case FETCH_RANDOM_GREETING_SUCCESS:
+      return action.payload;
+    default:
+      return state;
+  }
+};
+
+const rootReducer = combineReducers({
+  greeting: greetingReducer,
+});
```

Comment:
- You have done well so far, though after resolving the CORS error from the backend, your project works fine, in the microverse curriculum, it is better to use the `redux-toolkit` as this helps to prevent a lot of boilerplate code that comes with using the `redux` library, making our code concise and easy to read/understand. I suggest you use the [createSlice](https://redux-toolkit.js.org/api/createSlice) function to store your app's state. Please fix this 👍 👍